Library/Homebrew/install.rb

```diff
@@ -315,6 +318,21 @@ def install_formulae(
         end
       end.compact
 
+      if dry_run
+        formulae_name_to_install = formulae_to_install.map(&:name)
+        if formulae_name_to_install.present?
+          plural = "package".pluralize(formulae_name_to_install.count)
+          ohai "Would install #{formulae_name_to_install.count} #{plural}:"
+          puts formulae_name_to_install.join(" ")
```

Comment:
This is the same for casks with a different variable name, I wonder if they should be combined? If not, I think this should maybe use `formula` instead of `package` and the cask case `cask` instead of `package`.